backend/plortal/src/main/java/com/plortal/plortal/controller/ScheduleNotesController.java

```diff
@@ -44,11 +44,11 @@ public ResponseEntity<List<ScheduleNotes>> getAllScheduleNotes() {
     }
 
     @GetMapping("/api/v1/schedulenotes/date")
-    public ResponseEntity<List<ScheduleNotes>> getNotesByDate(@RequestParam String date) {
+    public ResponseEntity<List<ScheduleNotes>> getNotesByDate(@RequestParam String date,String email) {
         LocalDate parsedDate = LocalDate.parse(date);
         String dateString = parsedDate.toString();
         try {
-            List<ScheduleNotes> notes = service.findByDate(dateString);
+            List<ScheduleNotes> notes = service.findByDate(dateString,email);
```

Comment:
refactor - change service.findByDate to service.findByDateAndEmail and format files ctrl + alt + l